runtime/termscounter_test.go

```diff
@@ -0,0 +1,60 @@
+package runtime_test
+
+import (
+	"strings"
+	"testing"
+	"time"
+
+	"go.etcd.io/gofail/runtime"
+)
+
+var __fp_ExampleString *runtime.Failpoint = runtime.NewFailpoint("runtime_test", "ExampleString")
+
+func TestTermsCounter(t *testing.T) {
+	testcases := []struct {
+		name string
+		desc string
+		run  int
+		want string
+	}{
+		{
+			name: "runtime_test/ExampleString",
```

Comment:
Could you add more tests scenarios? For example: Before enable, after disable, after changing term etc.

Reply:
done